packages/cli/package.json

```diff
@@ -88,6 +103,7 @@
 		"compression-webpack-plugin": "^4.0.0",
 		"console-clear": "^1.0.0",
 		"copy-webpack-plugin": "^5.0.4",
+		"create-require": "^1.0.2",
```

Comment:
I'm not sure we need this. v10 is about to be EOL and `createRequire` is >= 12.2. I'm quite tempted to say that's fine?

Reply:
The `package.json` claims to support `node >= 8` https://github.com/preactjs/preact-cli/blob/fea6647982b639044130bbcc15a85ff8878a7388/packages/cli/package.json#L28-L30
As soon as it's over v8.9 I can just use `require.resolve`

Reply:
Ah, wasn't entirely sure what you were doing with it, just looked up the dep.

Anyways, you can ignore that, that's outdated and really needs to be updated. https://github.com/preactjs/preact-cli/pull/1493#issuecomment-744227337

Though that comment does say v12.0, so I take back what I said about the 12.2. I thought it was just 12.x

Reply:
Alright, `create-require` has been removed